Partition (WBP)/Functions/VI0039C.VTI0039.fun.yml

```diff
@@ -0,0 +1,71 @@
+﻿Function:
+  Enabled For RDMLX:  Yes
+
+  Source: |
+    Function Options(*DIRECT) Rcv_List(#wLKData #wLRet)
+    
+    * ******************************************************************************************************
+    * Test Case Description
+    * Issue #39 Programmer Commitment Control
+    * https://github.com/lansa/db-regression/issues/39
+    * ******************************************************************************************************
```

Comment:
You need to add in a summary of what the Function does based on a summary of what the ticket describes and what you have ended up implementing